### DISCUSSION TITLE
Breaking a cluster of two pools?

### DISCUSSION THREAD

**Discussion by BlackWand1** [minio/minio#18833] [Q&A] [answered]
If we have a cluster with 2 pools. What would happen if we simply broke them apart?

When we joined the second pool, we did not balance the storage. So all new data was uploaded to the new pool.
And if we brake them apart, would that work? But all the data in the old pool would just be missing? (just as if we removed that data)?

**Comment by klauspost** [comment]
Technically it can be done, but there is no method to do so and will require modifying internal state.

If you are actually looking for decommissioning the docs for it is here: https://min.io/docs/minio/linux/operations/install-deploy-manage/decommission-server-pool.html?ref=docs-internal

**Reply by BlackWand1** [reply]
OK. So if I just remove the other partner from the definitions "server{1...3}  it wont start?

**Reply by klauspost** [reply]
Correct. It will know that something is missing. 

To remove servers there is a process that will drain the old pool so it can be removed: https://min.io/docs/minio/linux/operations/install-deploy-manage/decommission-server-pool.html?ref=docs-internal

**Reply by BlackWand1** [reply]
Yeah, decomission has been started. Temporarily cancelled. But we are back to the performance problem. Where the old pool is affected the new pool, because you still need access to the old to list files etc. And it is performing soo bad it is a big problem.

So I was just trying to see if this was a possible solution.

**Reply by harshavardhana** [reply]
You can remove the pools @BlackWand1 we allow breaking it. Just make sure to provide the right command line arguments which are same as before without two pools.

**Comment by BlackWand1** [comment]
Ok. But what would be the impact?
Since there is no database or such, just files. The files would just be in the pool they reside?

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by harshavardhana** [reply, marked as answer]
Everything in the pool stays in its respective pool and everything is business as usual.

Your application should know that it has files in two different clusters now .
